src/EnergyPlus/SetPointManager.cc

```diff
@@ -3761,14 +4051,14 @@ void VerifySetPointManagers(EnergyPlusData &state, [[maybe_unused]] bool &Errors
                     continue;
                 ShowWarningError(state,
                                  format("{} =\"{}\"",
-                                        managerTypeName(state.dataSetPointManager->AllSetPtMgr(SetPtMgrNum).SPMType),
+                                        managerTypeName[static_cast<int>(state.dataSetPointManager->AllSetPtMgr(SetPtMgrNum).SPMType)],
```

Comment:
since the managerTypeName was replaced, this line (and the following similar lines) is changed.